.github/workflows/python-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: Python Tests
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        pip install poetry
+        poetry install
+    - name: Run pytest with coverage and generate report
+      run: poetry run pytest --cov=tests --cov-report=xml tests/
+    - name: Upload coverage report as artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: coverage-report
+        path: coverage.xml
```

Comment:
For what do you need the coverage report as artifact? I'd remove this since the coverage report's results are accessable through the next step.

Reply:
yes, this isn't necessary anymore, was a leftover from pytest-cov-comment. Fixed in https://github.com/caas-team/py-kube-downscaler/pull/14/commits/5c08545f75b634d67d52b630c724a13adc0a57b8